docs/api-attr.rst

```diff
@@ -0,0 +1,242 @@
+API Reference for the ``attr`` Namespace
+========================================
+
+.. module:: attr
+
+
+Core
+----
+
+.. autofunction:: attr.s(these=None, repr_ns=None, repr=None, cmp=None, hash=None, init=None, slots=False, frozen=False, weakref_slot=True, str=False, auto_attribs=False, kw_only=False, cache_hash=False, auto_exc=False, eq=None, order=None, auto_detect=False, collect_by_mro=False, getstate_setstate=None, on_setattr=None, field_transformer=None, match_args=True, unsafe_hash=None)
```

Comment:
Does this actually work? I double-checked the docs and it looks like `.. function::` accepts a signature to render but `.. autofunction::` expects just the name.
```suggestion
.. function:: attr.s(these=None, repr_ns=None, repr=None, cmp=None, hash=None, init=None, slots=False, frozen=False, weakref_slot=True, str=False, auto_attribs=False, kw_only=False, cache_hash=False, auto_exc=False, eq=None, order=None, auto_detect=False, collect_by_mro=False, getstate_setstate=None, on_setattr=None, field_transformer=None, match_args=True, unsafe_hash=None)
```

Reply:
Aha.. So it looks like it works https://attrs--1080.org.readthedocs.build/en/1080/api-attr.html#attr.s, just undocumented in Sphinx.

Reply:
Yeah that always worked. Otherwise you get very ugly signatures for decorators that accept arguments.